ogre_mesh_viewer.py

```diff
@@ -434,20 +489,29 @@ def keyPressed(self, evt):
     def mousePressed(self, evt):
         if evt.clicks != 2:
```

Comment:
I would allow selection with a single click, as double click is not intuitive

Reply:
I was even thinking of having a sort of highlight on hover using a red color, like the submeshes